packages/request-dashboard/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@requestnetwork/create-request-form",
```

Comment:
Please rename to "@requestnetwork/invoice-dashboard"

Rejected alternatives:
* Invoice Table - Rejected because this is more than just a table. approve/payment capability

DoD:
- [ ] Rename the folder name
- [ ] Update the NPM package name